packages/blade/src/utils/makeAccessible/makeAccessible.d.ts

```diff
@@ -0,0 +1,278 @@
+import type { AccessibilityRole } from 'react-native';
+export { default } from './makeAccessible.web';
+
+// All the WAI-ARIA 1.1 role attribute values from https://www.w3.org/TR/wai-aria-1.1/#role_definitions
+type AriaRoles =
+  | AccessibilityRole
+  | 'alert'
+  | 'alertdialog'
+  | 'application'
+  | 'article'
+  | 'banner'
+  | 'button'
+  | 'cell'
+  | 'checkbox'
+  | 'columnheader'
+  | 'combobox'
+  | 'complementary'
+  | 'contentinfo'
+  | 'definition'
+  | 'dialog'
+  | 'directory'
+  | 'document'
+  | 'feed'
+  | 'figure'
+  | 'form'
+  | 'grid'
+  | 'gridcell'
+  | 'group'
+  | 'heading'
+  | 'img'
+  | 'link'
+  | 'list'
+  | 'listbox'
+  | 'listitem'
+  | 'log'
+  | 'main'
+  | 'marquee'
+  | 'math'
+  | 'menu'
+  | 'menubar'
+  | 'menuitem'
+  | 'menuitemcheckbox'
+  | 'menuitemradio'
+  | 'navigation'
+  | 'none'
+  | 'note'
+  | 'option'
+  | 'presentation'
+  | 'progressbar'
+  | 'radio'
+  | 'radiogroup'
+  | 'region'
+  | 'row'
+  | 'rowgroup'
+  | 'rowheader'
+  | 'scrollbar'
+  | 'search'
+  | 'searchbox'
+  | 'separator'
+  | 'slider'
+  | 'spinbutton'
+  | 'status'
+  | 'switch'
+  | 'tab'
+  | 'table'
+  | 'tablist'
+  | 'tabpanel'
+  | 'term'
+  | 'textbox'
+  | 'timer'
+  | 'toolbar'
+  | 'tooltip'
+  | 'tree'
+  | 'treegrid'
+  | 'treeitem'
+  | (string & Record<string, unknown>);
```

Comment:
question: why this?

Reply:
I copy pasted this union from the React d.ts file. We can remove it

We don't need this but since you asked what this does is, say you want a union to have specific set of values but also accept any arbitrary string value but if you simply do this: 

<img width="609" alt="Screenshot 2022-06-17 at 4 32 19 PM" src="https://user-images.githubusercontent.com/35374649/174286170-482582f0-ba76-453d-9949-0abf837aec79.png">

You will lose autocomplete, adding `string` in union collapsed the whole union to just string.


But the trick here is if you add`(string & {})` it will accept any value without error but also show autocomplete

<img width="601" alt="Screenshot 2022-06-17 at 4 35 19 PM" src="https://user-images.githubusercontent.com/35374649/174286577-400c67b7-05f2-49bf-a6d8-1f52457820bf.png">

Reply:
👀 ohoooo interesting, but why is that the case that `string` vanishes the autocomplete?

Reply:
'hello' | 'world' | string

If you put 'hello' | 'world' a literal type union and add `string` which is wider than both `hello` | `world` it will widen the whole union to just be `string` type. 

tldr; `string` type is wider than `hello` | `world` both literals.